js/app/dashboard/components/MapProxy.js

```diff
@@ -190,34 +191,28 @@ export default class MapProxy {
     const coords = [task.address.geo.latitude, task.address.geo.longitude]
     const latLng = L.latLng(task.address.geo.latitude, task.address.geo.longitude)
 
-    let popupComponent = this.taskPopups.get(task['@id'])
+    let popupComponent
 
     if (!marker) {
 
       marker = MapHelper.createMarker(coords, iconName, 'marker', color)
 
       const el = document.createElement('div')
-
-      popupComponent = React.createRef()
-
-      const cb = () => {
-        this.taskMarkers.set(task['@id'], marker)
+      const root = createRoot(el)
+
+      this.taskMarkers.set(task['@id'], marker)
+
+      marker.bindPopup(() => {
+        popupComponent = React.createRef()
+        root.render(
+          <LeafletPopupContent
```

Comment:
I suggest wrapping it in StrictMode to get extra checks: 

https://github.com/coopcycle/coopcycle-web/blob/ee252a4b13fd6086f03973a2c18cd39d7ff5918c/js/app/foodtech/dashboard/index.js#L80-L86

https://react.dev/reference/react/StrictMode

Reply:
OK

I thought strict mode was enabled by default in react18 developpment? but maybe i am wrong

Reply:
I wrapped it in strict mode

Reply:
Actually the drag n drop doesnt work at all in strict mode, so i deleted it